CHANGELOG.md

```diff
@@ -40,6 +40,9 @@
 - Updated default snapshot to have more than one PlayerCreator entity.
 - Fixed package dependencies.
 - Worker flag changes are propagated to the `ViewDiff`.
+- Added a `Improbable.Gdk.Core.Editor` asmdef.
```

Comment:
asmdef looks like a keyboard-mashing typo :>

Reply:
Make sure to put this following line in your ashdfpafgd